lib/screens/home.dart

```diff
@@ -237,6 +238,9 @@ class _HomeScreenState extends State<HomeScreen> with TickerProviderStateMixin {
                             : Icons.voice_over_off),
                         tooltip: AppLocalizations.of(context)!.textToSpeech,
                         onPressed: () {
+                          // TODO: need to figure out how to actually
+                          // send the text to this plugin, but manual is working
+                          // TextToSpeechPlugin.speak("Testing this code!");
```

Comment:
This is working however, I'm not 100% how you want the text fed to the plugin

Reply:
This is fine for now.